apstools/migration/spec2ophyd.py

```diff
@@ -372,8 +392,39 @@ def create_ophyd_setup(spec_config):
         print(f"{device.ophyd_config()}")
 
 
+def get_options():
+    """Handle command line arguments."""
+    global parser
```

Comment:
Codacy found an issue: [Global variable 'parser' undefined at the module level](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5874497)